src/main/java/ChattyChatBot.java

```diff
@@ -0,0 +1,132 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class ChattyChatBot {
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    private static final String BOT_NAME = "Chatty Chat Bot";
+
+    private static final String BYE_STRING = "bye";
+    private static final String LIST_STRING = "list";
+    private static final String DONE_STRING = "done";
+    private static final String TODO_STRING = "todo";
+    private static final String DEADLINE_STRING = "deadline";
+    private static final String EVENT_STRING = "event";
+
+    private static final String SPACE_SEPARATOR = " ";
+    private static final String DOT_CHARACTER = ".";
+    private static final String BY_STRING = "/by";
+    private static final String AT_STRING = "/at";
+
+    private static final String LINE_BREAK = "____________________________________________________________";
+
+    private static final String ADDED_TASK_CONFIRMATION = "Got it. I've added this task:";
+    private static final String TASK_SUMMARY_FIRST_HALF = "Now you have ";
+    private static final String TASK_SUMMARY_SECOND_HALF = " tasks in the list.";
+
+    public static void main(String[] args) {
+        sendWelcomeMessage();
+
+        List<Task> tasks = new ArrayList<>();
+
+        String userInput = "";
+        do {
+            userInput = SCANNER.nextLine();
+
+            System.out.println(LINE_BREAK);
+
+            // Solution below adapted from: https://stackoverflow
+            // .com/questions/5067942/what-is-the-best-way-to-extract-the-first-word-from-a-string-in-java
+            String[] array = userInput.split(SPACE_SEPARATOR, 2);
+            String action = array[0];
+
+            switch (action) {
+            case LIST_STRING:
+                listAllTasks(tasks);
+                break;
+            case DONE_STRING:
+                markTaskAsDone(tasks, array[1]);
+                break;
+            case TODO_STRING:
+                addToDoTask(tasks, array[1]);
+                break;
+            case DEADLINE_STRING:
+                addDeadlineTask(tasks, array[1]);
+                break;
+            case EVENT_STRING:
+                addEventTask(tasks, array[1]);
+                break;
+            case BYE_STRING:
+                sendByeMessage();
+                break;
+            default:
+                sendDefaultResponse();
+            }
```

Comment:
Switch-case is a wise choice in this situation instead of the lengthy if-else lines